### PR TITLE
SPM FIR basis files

### PR DESCRIPTION
**Pull request by AlexBowring**
This PR creates:
- SPM FIR basis files complete with excursion set image. 

Performed on  all subjects of ds-011 from [BIDS-compliant datasets](https://drive.google.com/folderview?id=0B2JWN60ZLkgkMGlUY3B4MXZIZW8&usp=sharing).

### CODE REVIEW

**Comment by cmaumet** [comment]
Hi @AlexBowring! Everything looks good except for the excursion set image that is missing. Can you re-run _just_ the "Results Report" step to create this image (so that I don't have to review the full analysis again). Once this is done this PR will be ready to merge.

**Comment by AlexBowring** [comment]
@cmaumet The excursion set is in there, it is called spmF_0001_thresh.nii. I imagine you were looking for spmT_0001_thresh, but this is because we used an F contrast.

**Comment by cmaumet** [comment]
Then, is the `batch.m` file outdated?

File: spm_FIR_basis/batch.m [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+matlabbatch{14}.spm.stats.results.conspec.titlestr = '';
+matlabbatch{14}.spm.stats.results.conspec.contrasts = Inf;
+matlabbatch{14}.spm.stats.results.conspec.threshdesc = 'none';
+matlabbatch{14}.spm.stats.results.conspec.thresh = 0.001;
+matlabbatch{14}.spm.stats.results.conspec.extent = 0;
+matlabbatch{14}.spm.stats.results.conspec.conjunction = 1;
+matlabbatch{14}.spm.stats.results.conspec.mask.none = 1;
+matlabbatch{14}.spm.stats.results.units = 1;
+matlabbatch{14}.spm.stats.results.print = 'pdf';
+matlabbatch{14}.spm.stats.results.write.none = 1;
```
Comment (cmaumet) [inline spm_FIR_basis/batch.m]: Looking at the batch.m file the excursion set image is not saved, yet it is available in the spm_FIR folder. I am wondering which one is outdated (the excursion set image or the batch script)?

**Comment by AlexBowring** [comment]
The batch was wrong, should be right now!

**Comment by cmaumet** [comment]
Thank you @AlexBowring! This is ready to merge.

**Pull request merged**